src/DeviceDiscoveryExtension/DDDevice.cs

```diff
@@ -0,0 +1,26 @@
+//
+// Custom methods for DDDevice
+//
+// Authors:
+//   Israel Soto (issoto@microsoft.com)
+//
+// Copyright 2022 Microsoft Corporation.
+//
```

Comment:
I think this is from a past PR, but should the copyright headers be updated?

Reply:
Good point.

Reply:
Fixed now.